keyboards/412_64/readme.md

```diff
@@ -0,0 +1,15 @@
+# 412-64
+
+![412-64](image_here)
```

Comment:
No images of this?

Reply:
Not as yet - Once the PCBs are in, images will be uploaded. Jumped the gun, didn't I?

Reply:
No, it's fine.  Just wanted to make sure that it wasn't an oversight.